cfme/infrastructure/host.py

```diff
@@ -234,6 +234,13 @@ def power_off(self):
         pow_btn('Power Off', invokes_alert=True)
         sel.handle_alert()
 
+    def get_power_state(self):
+        return self.get_detail("Properties", "Power State")
```

Comment:
If the given object implement load_details() then you just add SummaryMixin in the base classes :)